lang2/macro/tests/ui/fail/20-unknown-method-marker.rs

```diff
@@ -0,0 +1,25 @@
+#![feature(proc_macro_hygiene)]
+
+use ink_lang2 as ink;
+
+#[ink::contract(version = "0.1.0")]
+mod noop {
+    #[ink(storage)]
+    struct Noop {}
+
+    impl Noop {
+        #[ink(constructor)]
+        fn new(&mut self) {}
+
+        #[ink(message)]
+        fn noop(&self) {}
+
+        #[ink(unknown_marker)]
+        fn has_unknown_marker(&self) {}
+
+        #[ink(unknown_marker_2)]
```

Comment:
Do we want all unknown markers, or just the first one?

Reply:
In theory we want all but for now we only warn on the first one.